README.md

```diff
@@ -2,12 +2,15 @@
 
 Starter template for writing an application using [Bonsai].
 
-This repository implements an application on Ethereum utilizing Bonsai as a coprocessor to the smart contract application.
+This repository implements an application on Ethereum utilizing Bonsai as a [coprocessor] to the smart contract application.
 It provides a starting point for building powerful new applications on Ethereum that offload computationally intensive
-(or difficult to implement) tasks to be proven by the [RISC Zero] [zkVM], with verifiable results sent to your Ethereum contract.
+(or difficult to implement) tasks to be proven by the [RISC Zero] [zkVM], with [verifiable results] sent to your Ethereum contract.
```

Comment:
Would this revision be accurate? i.e., at this point, does the verification happen automatically? 
```suggestion
(or difficult to implement) tasks to be proven by the [RISC Zero] [zkVM], with [verified results] sent to your Ethereum contract.
```